doc/user/shared-content/rbac-sm/enable-rbac.md

```diff
@@ -3,8 +3,8 @@ If RBAC is not enabled, all users have <red>**superuser**</red> privileges.
 {{</ warning >}}
 
 By default, role-based access control (RBAC) checks are not enabled (i.e.,
-enforced) when turning on [password
-authentication](/security/self-managed/authentication/#configuring-password-authentication). To
+enforced) when using [Password or SASL
```

Comment:
Let's just make this `using [authentication](/security/self-managed/authentication/#configuring-authentication-type)` ... since if we were to use short code, it won't work in this case ... and  if we add something else, we don't need to modify it going forward.

I had used "password authentication" before ... to kind of underscore that that's the only one we offered at that time ... but, now, I think going more generic will work for us.